<?xml version="1.0"?>
<launch>
	<arg name="image_topic" default="/top_front_camera/image_rect"/>

	<arg name="gpu_id" default="0"/>
	<arg name="model" default="mobilenet_thin"/>
	<arg name="resolution" default="656x368"/>

	<node name="pose_estimator" pkg="tfpose_ros" type="broadcaster_ros.py" output="screen" required="true">
		<env name="CUDA_VISIBLE_DEVICES" value="$(arg gpu_id)"/>
		<param name="camera" value="$(arg image_topic)"/>
		<param name="model" value="$(arg model)"/>
		<param name="resolution" value="$(arg resolution)"/>
	</node>

</launch>
